.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_size = 4
+end_of_line = lf
+
+[*.{c,h,cxx,am,m4}]
+indent_style = tab
```

Comment:
Seems its spelled `indent_size` :)  And I'd be surprised if there was no way to change the indentation size when using spaces.  Though, there indeed might be a setting to select the tab width, which might be different from the indentation step size.